.github/workflows/main-promote-builder-image.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     name: Promote Latest tag to cso Builder Image
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/sovereigncloudstack/cso-builder:1.0.0
+      image: ghcr.io/sovereigncloudstack/cso-builder:1.0.1
```

Comment:
the update looks wrong, we don't have this builder image present as of now. DNM. 
(On Go side, it's good.)

Reply:
This can be merged now.